cmd/client_changefeed.go

```diff
@@ -336,10 +336,13 @@ func verifyChangefeedParameters(ctx context.Context, cmd *cobra.Command, isCreat
 		}
 	}
 	switch sortEngine {
-	case model.SortUnified, model.SortInMemory, model.SortInFile:
+	case model.SortUnified, model.SortInMemory:
+	case model.SortInFile:
+		// obsolete. But we keep silent here. We create a Unified Sorter when the owner/processor sees this option
+		// for backward-compatibility.
 	default:
 		return nil, errors.Errorf("Creating changefeed with an invalid sort engine(%s), "+
```

Comment:
the `+` here is just for more readable ?

Reply:
Yes